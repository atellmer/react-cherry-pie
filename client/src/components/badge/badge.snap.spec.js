import React from 'react';
import renderer from 'react-test-renderer';

import TmBagde from './index';

jest.mock('react-dom');

test('Jest: TmBagde (Snapshot)', () => {
  const component = renderer.create(<TmBagde count={5}/>);
  let tree = component.toJSON();

  expect(tree).toMatchSnapshot();
});
