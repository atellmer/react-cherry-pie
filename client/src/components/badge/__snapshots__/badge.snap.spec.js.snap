exports[`test Jest: TmBagde (Snapshot) 1`] = `
<div
  className={undefined}>
  5
</div>
`;
